src/main/java/com/ashkan/moviesapi/MoviesAPI.java

```diff
@@ -0,0 +1,23 @@
+package com.ashkan.moviesapi;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+
+@SpringBootApplication
+
+public class MoviesAPI {
+
+    @Bean
```

Comment:
This is totally fine, but as a suggestion is better to use a different class to define the Beans you will need.
You can create a "configuration/Mapper.class" that will contain the definition of this Bean.

Reply:
Recomendation taken for beans, but this bean is not used ,so it will be deleted